api/proto/scheduler/scheduler.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+enum RunnerType {
+  DOCKER = 0;
```

Comment:
```suggestion
  Container = 0;
```

Reply:
we decided to stick to the same enum as the controller API